portfolio/src/main/webapp/index.html

```diff
@@ -59,11 +59,19 @@ <h3>Socials</h3>
       <p>GitLab: <a href="https://gitlab.com/sammurraytuesta">https://gitlab.com/sammurraytuesta</a></p>
       <h3>Gallery</h3>
       <img class="image" src="images/mini-me-hose.jpg"/>
```

Comment:
`<figure>` and `<figcaption>` are specifically intended for this use case of having an image and a associated caption. Consider using them instead.

Reply:
Thank you! I have made that change and learned something new!